src/Universum/Debug.hs

```diff
@@ -62,22 +63,27 @@ error s = P.error (unpack s)
 -- | Version of 'Debug.Trace.traceShow' that leaves warning.
 {-# WARNING traceShow "'traceShow' remains in code" #-}
 traceShow :: P.Show a => a -> b -> b
-traceShow a b = trace (P.show a) b
+traceShow a b = trace (pack (P.show a)) b
 
 -- | Version of 'Debug.Trace.traceShow' that leaves warning.
 {-# WARNING traceShowId "'traceShowId' remains in code" #-}
 traceShowId :: P.Show a => a -> a
-traceShowId a = trace (P.show a) a
+traceShowId a = trace (pack (P.show a)) a
+
+-- | Version of 'Debug.Trace.traceId' that leaves warning and takes 'Text'.
```

Comment:
This comment looks not very accurate.

Reply:
Or better just this one.

Reply:
I guess I wrote it because this function doesn't take `Text`.